substrate/frame/revive/src/limits.rs

```diff
@@ -58,3 +61,85 @@ pub const STORAGE_KEY_BYTES: u32 = 128;
 ///
 /// The buffer will always be disabled for on-chain execution.
 pub const DEBUG_BUFFER_BYTES: u32 = 2 * 1024 * 1024;
+
+/// The page size in which PolkaVM should allocate memory chunks.
+pub const PAGE_SIZE: u32 = 4 * 1024;
```

Comment:
maybe just use VM_MIN_PAGE_SIZE from polkavm-common?

Reply:
That constant's not for public consumption. (: (In general everything is 'polkavm-common' is private)

Besides, what matters is the page size that's configured, not what the minimum supported is, so this way is how it is actually intended to be used.